project/ProjectDependencies.scala

```diff
@@ -22,6 +22,8 @@ object ProjectDependencyVersions {
   val Log4jVersion = "2.5"
   val ProtobufVersion = "2.5.0"
   val SparkVersion = "1.6.1"
+  val VertxVersion = "3.0.0"
+  val ExampleVertxVersion = "3.3.2"
```

Comment:
Assuming that 3.3.2 is backwards-compatible to 3.0.0, why not depending on 3.3.2 in the adapter then?